site/scss/catalog/_settings.scss

```
@@ -15,3 +15,13 @@ $moo-catalog-font-family: "Geist" !default;
 // than the product component shadow scale. Keep the literal at the Sass
 // definition site so catalog.scss stays token-driven.
 $moo-catalog-command-shadow: 0 24px 50px -12px rgba(0, 0, 0, 0.25) !default;
+// The settings panel's theme-picker thumbnails mock the fixed light and
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)